guided-match/ddl.sql

```diff
@@ -51,9 +51,10 @@ CREATE INDEX JOUR_IDX2 on JOURNEYS(parent_journey_id);
 CREATE TABLE search_domains (
   domain_modifier_id               SERIAL PRIMARY KEY,
   search_id                        INTEGER NOT NULL,

```

Comment:
From the emails - were we adding `domain_modifier` in here too? Apols if I misunderstood..

Reply:
We have domain_modifie_id which is a system generated primary key . Then we have modifier_journey_name which on reflection should have a unique constraint around it. I think this is the same as domain_modifier. Especially f you were expecting domain_modifier to be a string. Then journey_selection_text will be used on the screen as perhaps a heading or title and journey_selection_description then further defines this.

Reply:
Yep ok, thought they might be one and the same thing - cheers